packages/react/src/Pillar/Pillar.features.stories.tsx

```diff
@@ -199,3 +199,48 @@ export const StackedWithLink: StoryFn<typeof Pillar> = () => {
     </Stack>
   )
 }
+
+export const FrostedGlassEffect: StoryFn<typeof Pillar> = () => {
```

Comment:
The colour contrast of the grey description text in this example drops quite low in places (2.9:1 on the left-most Pillar when over the dark-blue).

Of course this is only an example of component usage, but I think it might still be worth thinking about.

A few of the a11y issues that were raised in the recent audit related to inaccessible examples in our docs and storybook. Essentially, the goal is that all of our examples should promote accessibility best-practises since consumers may use our examples to inform their own designs and decisions. I'm not suggesting that anyone will copy this particular background image (hopefully!) but I do think that using a background image which doesn't result in areas of low contrast would allow us to lead by example.

The darker the background is, the lower the contrast drops, so maybe a vibrant, light image would be a better choice.

wdyt?

Reply:
Yeah, this is a fair point. That background is a stress test for internal use (VRT). It's not intended to be  end-user documentation. I'm wondering if we should have another 'subfolder' called tests / experiments, which auditors know to ignore. Ideally they wouldn't look at Storybook at all, as the public docs is our store front.

Reply:
Separately to the above, I've updated the pillar example now to use the same animated background as testimonial. It achieves the same thing from a VRT perspective and is more consistent with how we expect end-users to utilize the component.